src/Esfa.Vacancy.Register.Api/Web.config

```diff
@@ -15,6 +15,7 @@
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
     <add key="ApprenticeshipProviderElements" value="20" />
     <add key="LoggingRedisKey" value="logstash" />
+    <add key="DAS_AVMSPLUSCONNECTIONSTRING" value="" />
```

Comment:
you can remove the DAS_ here that's just used on environment variables

currently your env variable would be called

```
DAS_DAS_AVMS...
```